gulp/tasks/copy.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+var gulp = require( 'gulp' );
+var changed = require( 'gulp-changed' );
+var replace = require( 'gulp-replace' );
```

Comment:
I'm actually considering using the gulp auto-import plugin which would remove the requires anyway. Unless anyone has objections.